Detectors/ITSMFT/ITS/tracking/src/Tracker.cxx

```diff
@@ -64,12 +65,21 @@ void Tracker::clustersToTracks(std::function<void(std::string s)> logger, std::f
       error("Too much memory used during cell finding, check the detector status and/or the selections.");
       break;
     }
+    cellsPerCluster = mTimeFrame->getNumberOfClusters() > 0 ? float(mTimeFrame->getNumberOfCells()) / mTimeFrame->getNumberOfClusters() : 0.f;
+    if (mNumberOfRuns > 9 && cellsPerCluster > mTrkParams[iteration].CellsPerClusterLimit * mAverageCellsPerClusters) {
```

Comment:
Is clustersToTracks call meant per whole TF (with possibly preselected ROFs for reject)? In PbPb we can have quite large variation of multiplicity, which may also change the non-linear cells/clusters. Should not we have a cut on the abs. fraction of cells/cluster (then, if you want, also very relaxed cut of its factor wrt average)? Note that this protection will not be effective if the anomalous TF happens to the within first 9 iterations.

Also, if I get it right, the iteration is done with a particular set of cuts after suppressing the clusters of the previous iteration, right? Then each level iteration may have quite different cellsPerCluster. Should not per-iteration-type mAverageCellsPerClusters be calculated?
Finally, you apply the cut on the cell level, but already for the trackleting the time jumps by factor 100. Should we apply cut on this level?